src/main/kotlin/no/nav/pam/annonsemottak/scheduler/DefaultStillingOutboxSchedulerTask.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.pam.annonsemottak.scheduler
+
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock
+import no.nav.pam.annonsemottak.outbox.StillingOutboxService
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+
+interface StillingOutboxSchedulerTask {
+    fun prosesserStillingOutboxMeldinger()
+}
+
+@Component
+@ConditionalOnProperty(value = ["outbox.scheduler.enabled"], havingValue = "true")
+open class DefaultStillingOutboxSchedulerTask @Autowired constructor(
```

Comment:
Clue-et for å få dette til å fungere med en Kotlin-klasse var å innføre dette interface-et, og aktivere AspectJ (`@EnableAspectJAutoProxy`) i stede for CGLIB for proxy-er.